### DISCUSSION TITLE
Is it really useful to have this application?

### DISCUSSION THREAD

**Discussion by STVGecko** [FossifyOrg/General-Discussion#587] [Q&A]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Thank-You/issues) or [closed](https://github.com/FossifyOrg/Thank-You/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Thank-You/discussions) or [closed](https://github.com/FossifyOrg/Thank-You/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Thank-You/blob/main/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Wouldn't it be better to put all the functions unlocked by this application directly into the other applications?

### Why do you want this feature?

Is it really useful to have this application?

**Comment by naveensingh** [comment]
Thank You is intended to serve as a "donation" from Google Play users so yes, I think it's useful.

> unlocked by this application

As of now, there are no "locked" features in any Fossify app and I want to keep it that way. Thank You just extends all Fossify apps by adding one or two small features.

> put all the functions .. directly into the other applications

For technical reasons, you still need a separate app for global theming/preferences.

**Reply by STVGecko** [reply]
> Thank You is intended to serve as a "donation" from Google Play users so yes, I think it's useful.
> 
> > unlocked by this application
> 
> As of now, there are no "locked" features in any Fossify app and I want to keep it that way. Thank You just extends all Fossify apps by adding one or two small features.
> 
> > put all the functions .. directly into the other applications
> 
> For technical reasons, you still need a separate app for global theming/preferences.

Thanks for the reply, does the 'thank you' app allow you to adjust for global theming/preferences?

**Reply by naveensingh** [reply]
Right now it has the following features:

 - The ability synchronize a selected theme across all Fossify apps.
 - Shows the list of installed Fossify apps. Additionally, it shows apps that are trying to impersonate Fossify apps (simply any Fossify app that doesn't have the official signature is considered unsafe)
 - Accessibility option to enable checkmarks on switches in all apps. See https://github.com/orgs/FossifyOrg/discussions/78

**Reply by STVGecko** [reply]
> Right now it has the following features:
> 
>     * The ability synchronize a selected theme across all Fossify apps.
> 
>     * Shows the list of installed Fossify apps. Additionally, it shows apps that are trying to impersonate Fossify apps (simply any Fossify app that doesn't have the official signature is considered unsafe)
> 
>     * Accessibility option to enable checkmarks on switches in all apps. See https://github.com/orgs/FossifyOrg/discussions/78

Thank you for your answers. Sincerely 😊

**Reply by PeterKW** [reply]
Is there a compiled version of this to install? On FDroid or Google play?

**Reply by naveensingh** [reply]
It is available only on Google Play: https://github.com/orgs/FossifyOrg/discussions/57#discussioncomment-10965221

Right now it won't work with all apps until they are updated so there's no point in doing a Github release yet (for those who want it for the features). There are no plans to officially release Thank You on F-Droid.
